src/mavsdk/CMakeLists.txt

```
@@ -15,6 +15,7 @@ target_link_libraries(mavsdk
     PRIVATE
     JsonCpp::JsonCpp
     tinyxml2::tinyxml2
+    ssl
```

Does the target work (see [here](https://cmake.org/cmake/help/latest/module/FindOpenSSL.html#imported-targets))?

```suggestion
    OpenSSL::SSL
```